spec/factories/early_allocations.rb

```diff
@@ -18,6 +18,10 @@
 
     association :case_information
 
+    trait :eligible do
+
+    end
```

Comment:
Adding this trait a nice touch – I like it. But it might confuse people if they don't realise that the 'default' state of this factory is to be eligible, and therefore this trait needs to do 'nothing'.

It might be nice to add a simple one-liner comment in here to explain why this trait changes nothing.

Something along the lines of "Do nothing – eligible is the default outcome" should do the trick.